UpdateCgnOrchestrator/index.ts

```diff
@@ -114,19 +96,16 @@ export const handler = function*(
     const updateCgnResult = ActivityResult.decode(
       updateStatusResult
     ).getOrElseL(e =>
-      trackExAndThrow(e, "cgn.update.exception.decode.activityOutput")
+      trackExAndThrowWithError(e, "cgn.update.exception.decode.activityOutput")
     );
 
     if (updateCgnResult.kind !== "SUCCESS") {
-      trackExAndThrow(
+      trackExAndThrowWithError(
         new Error("Cannot update CGN Status"),
         "cgn.update.exception.failure.activityOutput"
       );
     }
 
-    // keep tracking of UserCgn update successfully
-    context.df.setCustomStatus("UPDATED");
-
     if (newStatusCard.status === ActivatedStatusEnum.ACTIVATED) {
```

Comment:
This branch is in a fire&forget logic, so it will not set `ERROR` custom status if it fails :)